casper/src/main/resources/PoS.rhox

```diff
@@ -76,27 +81,27 @@ new  PoS,
                 runMVar!(*stateCh, *processCh, *returnCh) |
 
                 getUser!(*userCh) |
-                for(@(rewards, bonds), resultCh <- processCh;
+                for(@(pending, rewards, bonds), resultCh <- processCh;
                     @userPk <- userCh) {
                   if (bonds.contains(userPk)) {
-                    resultCh!((rewards, bonds), (false, "Public key is already bonded."))
+                    resultCh!((pending, rewards, bonds), (false, "Public key is already bonded."))
                   } else if (amount < $$minimumBond$$) {
-                    resultCh!((rewards, bonds), (false, "Bond is less than minimum!"))
+                    resultCh!((pending, rewards, bonds), (false, "Bond is less than minimum!"))
                   } else if (amount > $$maximumBond$$) {
-                    resultCh!((rewards, bonds), (false, "Bond is greater than maximum!"))
+                    resultCh!((pending, rewards, bonds), (false, "Bond is greater than maximum!"))
                   } else {
                     deposit!(userPk, amount, *depositCh) |
                     for(@depositResult <- depositCh) {
                       match depositResult {
                         (true, _) => {
                           resultCh!(
-                            (rewards.set(userPk, 0), bonds.set(userPk,amount)),
+                            (pending, rewards.set(userPk, 0), bonds.set(userPk,amount)),
```

Comment:
Now, after debugging some situation where the keys are not found and the Map.get fails silently I would keep the pending code as it is and change the rewards code to use getOrElse because relying on get is more brittle than using getOrElse. What do you think?

Reply:
I have no preference as long as the usage is consistent.

Reply:
ok. I use getOrElse for both